src/i18n/en.js

```diff
@@ -57,6 +57,7 @@ export default {
   forgot_password: {
     back_login: `Did you remember your Password? Go back to Log in.`,
     back_login_after_forgot: `Back to Log in`,
+    blocked_account: `Your account has been cancelled. To know more please `,
```

Comment:
> `[...] To know more please `

Is that message broken? what happens with the rest of the content?

Reply:
yes html missing, need to change